src/test/regress/sql/citus_drain_node.sql

```diff
@@ -0,0 +1,62 @@
+CREATE SCHEMA citus_drain_node;
+SET search_path TO citus_drain_node;
+SET citus.shard_count TO 4;
+SET citus.shard_replication_factor TO 1;
+SET citus.next_shard_id TO 974653;
+
+SET client_min_messages TO ERROR;
+
+SELECT * FROM citus_set_coordinator_host('localhost', :master_port);
+SELECT * FROM master_set_node_property('localhost', :master_port, 'shouldhaveshards', true);
+
+CREATE TABLE test (x INT, y INT);
+SELECT create_distributed_table('test','x');
+
+CALL citus_cleanup_orphaned_shards();
+
+SELECT nodename, nodeport
```

Comment:
It's nice to add a count(*) here as well, to double check we're not losing placements.